Formula/rex.rb

```diff
@@ -11,7 +12,14 @@ class Rex < Formula
     sha256 "d02002fba499b2a3ed5271ccf1c3ddbaf75629d747aed289a4f53de7aad9d081" => :yosemite
   end
 
-  depends_on :perl => "5.16"
```

Comment:
There's no "maximum perl version" special handling, and upstream hasn't added support for the newest version, and it causes problems.

Theoretically under Homebrew's don't unnecessarily dupe the system dance it really shouldn't be using our Perl by default anyway, although I'm aware we intentionally violate this rule in places.